test/service/ConfigLoader.test.ts

```diff
@@ -61,6 +63,114 @@ describe('ConfigLoader', () => {
         expect(presetData).to.not.be.undefined;
     });
 
+    it('ConfigLoader custom maxUnlockedAccounts', async () => {
```

Comment:
how is this test related to the rest of this change?

Reply:
It's testing that when recreating the `presetData` without providing the custom preset, the cached custom preset is used (and the user-provided maxUnlockedAccounts is kept for example).